readme.md

```diff
@@ -1,7 +1,9 @@
 # Stars block
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://PaulMaistrenko
+.github.io/layout_stars/)
+- [TEST REPORT LINK](https://PaulMaistrenko
+.github.io/layout_stars/report/html_report/)
```

Comment:
Good job, but links doesn't work did you do deploy?

Reply:
yes, sure. I try fix it.

Reply:
My links alredy work